apps/wallet-mobile/src/features/Swap/useCases/StartSwapScreen/CreateOrder/EditPool/ShowPoolActions.tsx

```diff
@@ -35,7 +38,7 @@ export const ShowPoolActions = () => {
   const totalAmount = Quantities.format(amounts.buy.quantity, buyTokenInfo.decimals ?? 0)
   const protocolCapitalize = selectedPool.provider[0].toUpperCase() + selectedPool.provider.substring(1)
```

Comment:
can we avoid index accessing? maybe lodash has a function to handle capitalize already.